src/wwwroot/partials/settings/domain-manager.html

```diff
@@ -69,7 +69,7 @@ <h2 class="width--full">{{'domain-manager_title' | translate}}</h2>
             <svg class="domain--erase-button icon action-icon" ng-class="!vm.isDefaultDomain(d) || d.disabled ? 'default':'disabled'" ng-click="!vm.isDefaultDomain(d) && vm.deleteDomain(d)">
               <use xlink:href="/images/sprite.svg#doppler-icon-delete-icon"></use>
             </svg>
-            <a class="domain--dkim-info-action" target="_blank" href="#/settings/dkim-configuration-help?lang={{vm.langSelected}}&d={{d.name}}&sel={{d.dkim_selector}}&key={{d.dkim_public_key | escapeURI}}&dkim_status={{d.dkim_ready}}&spf_status={{d.spf_ready}}">
+            <a class="domain--dkim-info-action" target="_blank" href="#/settings/dkim-configuration-help?lang={{vm.langSelected}}&d={{d.name}}">
```

Comment:
It is not part of your task, but I think that you can remove _lang_ parameter because it is not required in this context.